age--1.4.0.sql

```diff
@@ -3062,9 +3111,9 @@ PARALLEL SAFE;
 CREATE OPERATOR CLASS ag_catalog.gin_agtype_ops
 DEFAULT FOR TYPE agtype USING gin AS
   OPERATOR 7 @>,
-  OPERATOR 9 ?(agtype, text),
-  OPERATOR 10 ?|(agtype, text[]),
-  OPERATOR 11 ?&(agtype, text[]),
+  OPERATOR 9 ?(agtype, agtype),
```

Comment:
Why the change to agtype for the last arg?

Reply:
@jrgemignani In cypher queries, only agtype is supported. So it had to be changed from text to agtype so that these operators can be used by gin.

Reply:
ty